src/graph/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const NODE_HEIGHT = 125
+export const NODE_WIDTH = 205
+
+export const NODE_HEIGHT2 = 320
+export const NODE_WIDTH2 = 480
```

Comment:
For clear let's `NODE_HEIGHT2` and `NODE_WIDTH2` rename into `EXPANDED_NODE_HEIGHT` and `EXPANDED_NODE_WIDTH`

Reply:
Thank you. I have now completely removed the expand feature. It is likely not going to help us implement node grouping anyway